catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/metacard/AbstractBasicMetacardUpdater.java

```diff
@@ -28,7 +30,7 @@ protected AbstractBasicMetacardUpdater(String attributeName) {
     }
 
     @Override
-    public final void update(Metacard parent, Metacard child) {
+    public final void update(Metacard parent, Metacard child, Context context) {
```

Comment:
Interesting article on this topic from Brian Goetz, Java language architect: http://www.ibm.com/developerworks/library/j-jtp1029/